src/main/java/com/coherent/training/selenium/kapitsa/web/base/BrowserDriverFactory.java

```diff
@@ -22,7 +23,6 @@ public class BrowserDriverFactory {
     private static final String EDGE_DRIVER = "src/test/resources/drivers/msedgedriver.exe";
     private static final String DOWNLOAD_FOLDER = "downloadFiles";
     private static final String PROJECT_DIR = System.getProperty("user.dir");
```

Comment:
let's make constant class with config keys => we'll have private static final String PROJECT_DIR = System.getProperty(PROJECT_DIR); where PROJECT_DIR = "user.dir"